IAMOTZ/ievents
# #153339820 Implement the home page with react

#### What does this PR do?
It uses react and redux to implement the home page of the application
#### Description of Task to be completed?
User should be able to see a home page
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started, the signup page should be loaded there
- the home page should open there
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
#### What are the relevant pivotal tracker stories?
#153339820
#### Screenshots (if appropriate)
#### Questions:
None
